.gitattributes

```diff
@@ -1,4 +1,6 @@
 .gitattributes export-ignore
 .gitignore export-ignore
 .gitmodules export-ignore
+.gitlab-ci.yml export-ignore
 archive.sh export-ignore
+circ-docker.tar.gz export-ignore
```

Comment:
maybe a more general term for this?

Reply:
That line is a legacy; it is from artifact generating script, which is not part of the benchmark. Therefore, I have removed it.